RedHat-UX/red-hat-design-system
# docs: images not rendering on patterns landing page

Closes #1070

## What I did

1. Added `srcAbsolute` to the example images shortcode to correct the image paths under /patterns
2. Updated images and summaries

## Testing Instructions

1. You should see images again in the [DP](https://deploy-preview-1071--red-hat-design-system.netlify.app/patterns/)
2. & they should match the [design spec](https://xd.adobe.com/view/0f123ffb-ae98-411e-ada3-2526129e0c74-5a41/)
